### ISSUE TITLE
[Feature request] Apnonce setter for devices stuck in recovery mode

### ISSUE THREAD

**Issue by stblr** [checkra1n/BugTracker#774]
I tried to update an iPhone SE from 12.4 to 13.3.1 using idevicerestore but it failed, maybe because of the lack of free space on the device. Now it's stuck on recovery mode and can't be updated using iTunes either (error 14 similar to libimobiledevice/idevicerestore#312). A clean restore would be a trivial solution but there is some data on the device I would prefer not to lose. I have 12.4 blobs so with a nonce setter I could "update" to it, do a backup and then clean restore to 13.3.1.

**Comment by Siguza** [comment]
While setting nonce might be useful, I don't think it would help in your case, because SEP wouldn't play along. I'll mark this as deferred, with the note that I think building a separate module on top of pongoOS might be the most viable and accessible thing to do...

**Comment by stblr** [comment]
Okay, thanks for answering.

So 12.4 is completely incompatible with 13.3.1 SEP, i.e. it will not just break Touch ID, but I won't be able to downgrade at all right?

**Comment by Siguza** [comment]
Yes. I know that any SEP above 13.1.3 is fully incompatible with iOS 12.x, but I suspect even 13.0 SEP would be.  
And attempting to restore with a fully incompatible SEP will cause the restore procedure itself to fail.

**Comment by parrotgeek1** [comment]
> I suspect even 13.0 SEP would be.

It is; even 13.0 beta 1 SEP was. I tried it last summer.
